packages/backend-common/src/service/lib/config.ts

```diff
@@ -47,14 +47,14 @@ export type CertificateReferenceOptions = {
 };
 
 export type CertificateSigningOptions = {
-  algorithm: string;
+  algorithm?: string;
   size?: number;
```

Comment:
just noticed: if i understand it correctly, config.d says keySize and so does selfsigned, but here it's just size - intentional?

Reply:
Probably not :grin:, this deprecates the entire thing though